staging/knative/PATCHES.md

```diff
@@ -11,7 +11,7 @@ When fetching `serving-core.yaml` from [upstream](https://github.com/knative/ser
 
 1. Get the latest file
 ```bash
-wget https://github.com/knative/serving/releases/download/knative-v1.9.2/serving-core.yaml
+wget https://github.com/knative/serving/releases/download/knative-v1.9.3/serving-core.yaml
```

Comment:
just wondering, but were the steps in this file used to generate the changes in this PR? do we need to update anything here to make sure the next person to do this is able to?

Reply:
@gracedo I might redo this file, because these steps are not always necessary. We use a very stripped-down version of knative-serving, which is only like 1/3 of the yaml you get from these steps. I might document my method, and leave this at the bottom to refer to if a complete redo is ever necessary.

Reply:
That would be great! To help the next person out :) (since it sounds like it was painful for you to figure out 😅 )